ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -1,6 +1,10 @@
 from styx_msgs.msg import TrafficLight
 import numpy as np
 import cv2
+from keras.models import model_from_json
+import tensorflow as tf
+import json
+import rospy
 
 class TLClassifier(object):
```

Comment:
I would suggest that instead of out-commenting old CV classifier, we put it in a separate class (in same module), say TLClassifierCV. This way in tf_detector.py we can have:

```python
self.light_classifier = TLClassifier()
# self.light_classifier = TLClassifierCV()
```
so that if we want to quickly disable 'real' classifier to only test rest of the pipeline, just one line needs to be changed.